.github/workflows/test.yml

```diff
@@ -13,6 +13,18 @@ on:
     - cron: '30 5 * * 1'
 
 jobs:
+  windows:
```

Comment:
This new job should be added to the test roll-up job at the end of this workflow:

```
  test:
    if: always()
    needs: [clippy, test-matrix, all-features, oldstable, windows]
```